rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -48,7 +48,8 @@ bool serialize<rsx::rsx_state>(utils::serial& ar, rsx::rsx_state& o)
 	ar(o.transform_program);
 
 	// Hack for compatiblity with previous RSX captures
-	if (rsx::get_current_renderer()->state & cpu_flag::exit || GET_SERIALIZATION_VERSION(global_version))
+	if (auto rsxthr = rsx::get_current_renderer();
+		GET_SERIALIZATION_VERSION(global_version) || (rsxthr && rsxthr->state & cpu_flag::exit))
```

Comment:
The current fix would be to check if the current thread if the current is the RSX.